modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -149,6 +154,58 @@ public Account getAccountByExternalReferenceCode(
 				_accountResourceDTOConverter, externalReferenceCode));
 	}
 
+	@Override
+	public Page<Account> getAccountGroupAccountsPage(
+			Long accountGroupId, String search, Filter filter,
+			Pagination pagination, Sort[] sorts)
+		throws Exception {
+
+		AccountGroup accountGroup = _accountGroupService.getAccountGroup(
+			accountGroupId);
+
+		return SearchUtil.search(
+			Collections.emptyMap(),
+			booleanQuery -> {
+				BooleanFilter booleanFilter =
+					booleanQuery.getPreBooleanFilter();
+
+				booleanFilter.add(
+					new TermFilter(
+						"accountGroupIds",
+						String.valueOf(accountGroup.getAccountGroupId())),
+					BooleanClauseOccur.MUST);
+			},
+			filter, AccountEntry.class.getName(), search, pagination,
+			queryConfig -> queryConfig.setSelectedFieldNames(
+				Field.ENTRY_CLASS_PK),
+			searchContext -> searchContext.setCompanyId(
+				contextCompany.getCompanyId()),
+			sorts,
+			document -> _toAccount(
+				Collections.emptyMap(),
+				GetterUtil.getLong(document.get(Field.ENTRY_CLASS_PK))));
+	}
+
+	@Override
+	public Page<Account> getAccountGroupByExternalReferenceCodeAccountsPage(
+			String externalReferenceCode, String search, Filter filter,
```

Comment:
externalReferenceCode -> accountGroupExternalReferenceCode (like the parameter in the yaml - and in the base class)